packages/eslint-plugin/src/rules/comma-spacing.ts

```diff
@@ -110,6 +110,7 @@ export default createRule<Options, MessageIds>({
       if (
         prevToken &&
         isTokenOnSameLine(prevToken, commaToken) &&
+        // eslint-disable-next-line deprecation/deprecation -- TODO - switch once our min ESLint version is 6.7.0
```

Comment:
You can use `eslint-plugin-unicorn` to ensure you fix these TODOs with a specific dependency version requirement.

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/expiring-todo-comments.md#dependency-version

Reply:
that's a really cool rule!

sadly I don't think this will work for us because our current dependency version on ESLint is v8.x, but this is about our minimum supported version of the package.